app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -80,8 +80,8 @@ def log_clone_options(clone_options)
     _log.info("Destination VM Name:        [#{clone_options[:name]}]")
     _log.info("Destination Host:           [#{clone_options[:host].name} (#{clone_options[:host].ems_ref})]")
     _log.info("Destination Datastore:      [#{clone_options[:datastore].name} (#{clone_options[:datastore].ems_ref})]")
-    _log.info("Destination Folder:         [#{clone_options[:folder].name}] (#{clone_options[:folder].ems_ref})")
-    _log.info("Destination Resource Pool:  [#{clone_options[:pool].name} (#{clone_options[:pool].ems_ref})]")
+    _log.info("Destination Folder:         [#{clone_options[:folder].name}] (#{clone_options[:folder].ems_ref})") if clone_options[:folder]
+    _log.info("Destination Resource Pool:  [#{clone_options[:pool].name} (#{clone_options[:pool].ems_ref})]") if clone_options[:pool]
```

Comment:
@gmcculloug `folder` is a required argument to [CloneVM_Task](https://www.vmware.com/support/developer/vc-sdk/visdk41pubs/ApiReference/vim.VirtualMachine.html#clone), and `resource_pool` is a required parameter to [VirtualMachineRelocateSpec](https://www.vmware.com/support/developer/vc-sdk/visdk41pubs/ApiReference/vim.vm.RelocateSpec.html) when cloning from a template to a VM.

Technically resource pool is optional which means by default it will keep the source's resource pool, but a template does not have a resource pool which is why it is required when cloning from a template.

>  For a clone operation from a template to a virtual machine, this argument is required.

>